packages/eslint-config-godaddy/package.json

```
@@ -3,37 +3,39 @@
   "version": "7.1.1",
   "description": "Base ESLint config for consistent style in ES6 projects at GoDaddy.",
   "license": "MIT",
+  "type": "module",
+  "main": "index.js",
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run --silent lint",
     "test": "echo ok",
-    "clean": "rimraf node_modules",
-    "unused": "eslint-find-rules --unused ./index.js || echo ''"
+    "clean": "rimraf node_modules"
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:godaddy/javascript.git"
+    "url": "git@github.com:godaddy/javascript.git",
+    "directory": "packages/eslint-config-godaddy"
   },
   "keywords": [
     "godaddy",
     "javascript",
     "styleguide",
     "style-guide",
-    "eslint",
-    "es6"
+    "eslint"
   ],
-  "bin": {
-    "eslint-godaddy": "bin/eslint-godaddy"
-  },
   "peerDependencies": {
-    "eslint": "^8"
+    "eslint": ">= 9"
```

**question (non-blocking):** I presume `npm` does not care about about the whitespace between `>=` and `9`, but just raising it here.

---

I recommend sticking to the major version range. Changes are there that will break and change again in a future 10.x.

```suggestion
    "eslint": "^9"
```